code/src/Components/Summary.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+
+export const Summary = (props) => {
+  const { name, radioInput, food } = props
```

Comment:
You can be consistent with how you retrieve the props:
`export const Summary = ({ name, radioInput, food }) => {`

Reply:
Ooo I see! Thank you very much for it!